solr/prometheus-exporter/src/test/org/apache/solr/prometheus/exporter/SolrExporterIntegrationTest.java

```diff
@@ -58,7 +58,7 @@ public void solrExporterDurationMetric() throws Exception {
     assertTrue(durationHistogram.get("solr_exporter_duration_seconds_sum") > 0);
 
     // 17 = (15 buckets in the histogram) + (count metric) + (sum metric)
-    assertEquals(17, durationHistogram.size());
+    assertEquals(18, durationHistogram.size());
```

Comment:
I have no idea why this histogram has one more entry, anyone familiar with this?

Reply:
I think this is `solr_exporter_duration_seconds_created`  but I don't know where it came from right now.

These are all the durationHistogram keys

```
[solr_exporter_duration_seconds_bucket{le="+Inf",}, 
solr_exporter_duration_seconds_bucket{le="0.005",}, 
solr_exporter_duration_seconds_bucket{le="0.01",}, 
solr_exporter_duration_seconds_bucket{le="0.025",}, 
solr_exporter_duration_seconds_bucket{le="0.05",}, 
solr_exporter_duration_seconds_bucket{le="0.075",}, 
solr_exporter_duration_seconds_bucket{le="0.1",}, 
solr_exporter_duration_seconds_bucket{le="0.25",}, 
solr_exporter_duration_seconds_bucket{le="0.5",}, 
solr_exporter_duration_seconds_bucket{le="0.75",}, 
solr_exporter_duration_seconds_bucket{le="1.0",}, 
solr_exporter_duration_seconds_bucket{le="10.0",}, 
solr_exporter_duration_seconds_bucket{le="2.5",}, 
solr_exporter_duration_seconds_bucket{le="5.0",}, 
solr_exporter_duration_seconds_bucket{le="7.5",},
solr_exporter_duration_seconds_count, 
solr_exporter_duration_seconds_created, 
solr_exporter_duration_seconds_sum]
```

Reply:
Ah bingo:

* https://github.com/prometheus/client_java/pull/615

which came in as part of https://github.com/prometheus/client_java/releases/tag/parent-0.10.0

There is https://github.com/prometheus/client_java/releases/tag/parent-0.16.0 which has https://github.com/prometheus/client_java/pull/791

* https://github.com/prometheus/client_java/tree/main#disabling-_created-metrics

I don't think this is an issue and just updating the count makes sense. The comment should be updated though.

Reply:
Thanks for the digging, makes sense!